_includes/syllabus.html

```diff
@@ -0,0 +1,75 @@
+{% comment %}
+  Display syllabus in tabular form.
+  Days are displayed if at least one episode has 'start = true'.
+{% endcomment %}
+<div class="syllabus">
+  <h2 id="schedule">Schedule</h2>
+
+  {% assign day = 0 %}
+  {% assign multiday = false %}
+  {% for episode in site.episodes %}
+    {% if episode.start %}{% assign multiday = true %}{% break %}{% endif %}
+  {% endfor %}
+  {% assign current = site.start_time %}
+
+  <table class="table table-striped">
+  <tr>
+    <td class="col-md-1"></td>
+    <td class="col-md-3"><a href="{{ page.root }}/setup">Setup</a></td>
+    <td class="col-md-7">Dowload files used on the lesson.</td>
```

Comment:
Yes, sorry, my english is getting worse and worse. I blame it on a typo. Will create the setup page.